src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -9426,7 +9426,7 @@ namespace HVACVariableRefrigerantFlow {
 
         // if the terminal unit and fan are scheduled on then set flow rate
         if (GetCurrentScheduleValue(state, VRFTU(VRFTUNum).SchedPtr) > 0.0 &&
-            (GetCurrentScheduleValue(state, VRFTU(VRFTUNum).FanAvailSchedPtr) > 0.0 || DataHVACGlobals::ZoneCompTurnFansOff) &&
+            (GetCurrentScheduleValue(state, VRFTU(VRFTUNum).FanAvailSchedPtr) > 0.0 || DataHVACGlobals::ZoneCompTurnFansOn) &&
```

Comment:
Interesting. So, following the blame back, this was broken back in January. So, there's the potential that this introduced a bug where an availability signal of `ZoneCompTurnFansOff` would actually turn on the VRFTU fan. A rare case, for sure, and clearly not in our regression suite, but we should probably rename the title and describe the potential problem in the PR description just in case someone runs across it in v9.3 or 9.4.

Reply:
Title and description are updated.